playground/src/components/state-machines-list/state-machines-list.js

```diff
@@ -38,6 +38,10 @@ const getErrorMessage = error =>
 export class StateMachinesList extends React.Component {
   static displayName = 'StateMachinesList';
   static propTypes = {
+    history: PropTypes.shape({
+      push: PropTypes.func.isRequired,
+    }).isRequired,
```

Comment:
Given the comment below: we might not have to share the history if we just pass down a fn `goToStateMachinesList` and `goToStateMachineDetail`. Hope it makes sense.

Reply:
Right, we can use `goToStateMachineDetail` in this case. I missed that, thanks!

Reply:
👍 
82e4aaa